app/pages/solutions-list/controller.test.js

```diff
@@ -91,6 +93,29 @@ describe('solutions-list controller', () => {
         expect(err).toEqual(new Error('No manifest found for filter type: unknown'));
       }
     });
+
+    it('should not call getData if the filterType is covid19', async () => {
+      createContext.createSolutionListPageContext
+        .mockResolvedValueOnce();
+
+      await getSolutionListPageContext({ filterType: 'covid19' });
+
+      expect(apiProvider.getData.mock.calls.length).toEqual(0);
+    });
+
+    it('should get data locally if the filterType is covid19 ', async () => {
```

Comment:
`should call createSolutionListPageContext with the local content if filterType is covid19`